### ISSUE TITLE
Databron fout

### ISSUE THREAD

**Issue by TVDD90** [rduivenvoorde/pdokservicesplugin#102]
De lagen die ik via de PDOK Plugin download, werken helaas niet door een databron error.
Dit geldt voor zowel reeds eerder toegevoegde lagen als nieuwe lagen. 
Is dit een bekend probleem? en is hier een oplossing voor?

Mvg. Thijs

**Comment by rduivenvoorde** [comment]
Yep, wordt aan gewerkt zie: https://github.com/rduivenvoorde/pdokservicesplugin/pull/101

**Comment by TVDD90** [comment]
Top, dank voor je reactie.

**Comment by rduivenvoorde** [comment]
Hoi Thijs,

Kun je even aangeven om welke lagen het gaat?
Want pdok verandert soms de service-url's van pdok lagen.
En dat betekent dat wij die dan ook in de plugin moeten aanpassen.
Vaak houdt pdok zo'n url nog een tijdje werkend, maar op enig moment trekken ze er de stekker uit.

Voor mij is het nu even belangrijk of jij:
- oude projecten gebruikt met daarin verbindingen naar oude niet meer bestaande pdok services 
(dat kunnen wij in de plugin niet fixen, je moet de lagen opnieuw opzoeken en aanmaken)
- er service url's zijn in de plugin die niet werken
(DAT moeten en kunnen wij wel aanpassen... maar dan moet ik precies weten welke niet zouden werken)

**Comment by TVDD90** [comment]
Bedankt voor je reactie. Dat vermoeden had ik al. 
Het gaat eigenlijk om alle lagen die ik importeer. Eerder deden alle lagen het wel gewoon.

![prntscr](https://github.com/rduivenvoorde/pdokservicesplugin/assets/135694724/a0315e5b-aabd-4e49-a979-afa0deb3cbf1)

**Comment by rduivenvoorde** [comment]
Ik kan het niet voldoende lezen, maar normaal idd als je een laag vanuit de plugin in je project laadt, dan slaat QGIS de 'service'-url van die laag op in je projectbestand.
Ik vind het jammer dat PDOK geen vaste url's gebruikt, ze rommelen er nogal veel mee vind ik.
Maar van de lagen die er nu niet meer zijn, moet je kijken of je ze weer in de pdokserviceplugin kan vinden en ze dan opnieuw laden (want dan heb je weer de juiste services url). Je zou ook de xml van het projectbestand kunnen openen en dan de url's handmatig aanpassen.

Ik sluit dit issue dan.
